Cargo.toml

```diff
@@ -37,12 +37,17 @@ push_macros = { path = "packages/push-macros" }
 
 
 [workspace.lints.clippy]
-pedantic = "warn"
-nursery = "warn"
-cargo = "warn"
-perf = "warn"
-complexity = "warn"
-correctness = "deny"
+# Lint groups
```

Comment:
Might it be nice to have a comment that says something about why all these have a priority of -1?

Presumably the reason we need to add this is because we have all the individual lints, right? I haven't chased all those down, but I presume there's not some group that would include all of them for us?

Reply:
Maybe some of them, but not all, no. The reason for the -1 is that that ensures that the individual lints (with a default priority of 0 ) override the groups, if they are different.

Reply:
I'll add a comment in the file explaining it.

Reply:
That's actually what I intended, but didn't say very clearly. Thanks!